appcues/src/main/java/com/appcues/action/ActionKoin.kt

```diff
@@ -14,7 +14,7 @@ internal object ActionKoin : KoinScopePlugin {
 
     override fun ScopeDSL.install() {
         scoped { ActionRegistry(scope = get(), logcues = get()) }
-        scoped { ActionProcessor(appcues = get(), appcuesCoroutineScope = get()) }
+        scoped { ActionProcessor(scope = get()) }
```

Comment:
not part of this PR, but the `logcues` thing on the ActionRegistry line above is similar - mix of injection patterns

Reply:
I will fix this as well